src/typing_manager.ts

```diff
@@ -52,6 +56,9 @@ export class TypingManager implements Disposable {
         this.disposables.push(commands.registerCommand("vscode-neovim.send", this.onSendCommand));
         this.disposables.push(commands.registerCommand("vscode-neovim.send-blocking", this.onSendBlockingCommand));
         this.disposables.push(commands.registerCommand("vscode-neovim.escape", this.onEscapeKeyCommand));
+        this.disposables.push(commands.registerCommand("vscode-neovim.toggle", this.onToggleCommand));
+        this.disposables.push(commands.registerCommand("vscode-neovim.enabled", this.onToggleCommand));
+        this.disposables.push(commands.registerCommand("vscode-neovim.disabled", this.onToggleCommand));
```

Comment:
pointing to the wrong function?

Reply:
thanks for the review!
it's changed now